Download.Mvc/Startup.cs

```diff
@@ -40,6 +41,8 @@ public void ConfigureServices(IServiceCollection services)
 
             // Add framework services.
             services.AddMvc();
+
+            services.AddSingleton<ITitleCodesService, TitleCodesService>();

```

Comment:
@jSylvestre Doesn't matter in this case, and thus AddSingleton works just fine and is maybe a bit more performant.  AddTransient is safe if we make changes in the future, but for now the thing to look at when deciding is what state does the service hold.  In this case, it holds no state that would be different between calls, thus a singleton is fine